x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/ShardFollowTask.java

```diff
@@ -36,7 +36,7 @@
     public static final Set<String> HEADER_FILTERS =
         Collections.unmodifiableSet(new HashSet<>(Arrays.asList("es-security-runas-user", "_xpack_security_authentication")));
 
-    static final ParseField LEADER_CLUSTER_ALIAS_FIELD = new ParseField("leader_cluster_alias");
+    static final ParseField LEADER_CLUSTER_FIELD = new ParseField("leader_cluster");
```

Comment:
I think that since we call this and other fields `leader_cluster`, the backing fields and getters/setters should reflect this name too (rather than `leader_cluster_alias`.

Reply:
Agreed, I completely missed this.